### DISCUSSION TITLE
þriggja ára afmæli reglunnar - 2024/2/0

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#188] [vefsida-ummæli]
# þriggja ára afmæli reglunnar - 2024/2/0

Í dag er þriggja ára afmæli reglunnar og mikil veisla

https://www.biskupsreglan.com/frettir/3-%C3%A1ra-afm%C3%A6li

**Comment by TJ478** [comment]
Þetta eru stórkostlegar fréttir og ég vona að framtíð reglunnar verði enn betri.
